client/src/components/DetailedRecipe.jsx

```diff
@@ -131,7 +143,18 @@ function DetailedRecipe({ editMode, recipeID }) {
             />
           </div>
         </div>
-        {editMode ? <Button text="Save" onClick={uploadRecipe} /> : null}
+        {editMode ? (
+          <Button
+            text="Save"
+            onClick={uploadRecipe}
+            addClassName={`text-white px-4 py-2 rounded focus:outline-none  ${
+              emptyFields
+                ? 'custom-button-color border custom-button-border custom-button-shadow text-white px-4 py-1 rounded focus:outline-none mx-1 grayscale disabled-button'

```

Comment:
szerintem ezek a custom-button-color, custom-button-border és társai nem létező classok, olyan mintha chatgpt ről lenne kimásolva az egész

Reply:
Ezt tailwind sort az oldalunkrol màsoltam ki. Inspector a szürke post recipe gombra ès ott volt. 
No chatgpt

Reply:
Nem tudom, hogy a Post recipe gombot ki csinàlta. Sztem ő tudja honnan van.azèrt màsoltam h egysèges legyen.

Reply:
A PostRecipe komponenst én csináltam, és nincs benne semmi ilyesmi. Sehol nincs a kódunkban ilyen.

Reply:
Ja én a komponensre gondoltam, nem a gombra ott fent. Látom ott is ez van. Az sem jó :/